packages/tss-client/src/client.ts

```diff
@@ -283,7 +253,7 @@ export class Client {
     });
   }
 
-  precompute(tss: typeof TssLib, additionalParams?: Record<string, unknown>) {
+  precompute(additionalParams?: Record<string, unknown>) {
```

Comment:
shall we change the name of variable from additionalParams to just params, since this its the only arg being passed in the function